docs/execute.adoc

```diff
@@ -37,10 +37,10 @@ image::diag/iqconfigs.png[Issue queue configurations,width=900,align=center,titl
  * The *unified* configuration features a single sequencer for all arithmetic operations fed by a VIQ. This configuration has the lowest hardware cost, but cannot simultaneously saturate integer and floating-point functional units.
  * The *shared* configuration features separate integer and floating-point sequencers fed by a common VIQ.
  Instructions are issued from the shared VIQ to either the integer or floating-point sequencer.
- This configuration can simultaneously saturate integer and floating-point execution units with well-scheduled code.
- * The *split* configuration features separate integer and floating-point sequencers each fed by an independent issue queue. This configuration is the most performant, as it can dynamically schedule out-of-order across integer and floating-point instructions, but is also the most costly, owing to the separate floating-point issue queue.
+ This configuration can simultaneously saturate integer and floating-point execution units with well-scheduled code, requiring the interleaving of integer and floating-point operations.
+ * The *split* configuration features separate integer and floating-point sequencers each fed by an independent issue queue. This configuration is the most performant and the most flexible, as it can dynamically schedule out-of-order across integer and floating-point instructions. It incurs additional area costs owing to the separate floating-point issue queue.
  * The *multi-ALU* configuration adds an additional integer sequencer and functional units to enable dual-issue of integer ALU operations
- * The *multi-MAC* and *multi-FMA* configurations add additional integer or floating point sequencers along with functional units to enable dual-issue of integer or floating-point multiply-accumulate operations, respectively
+ * The *multi-MAC* and *multi-FMA* configurations add additional integer or floating point sequencers along with functional units to enable dual-issue of integer or floating-point multiply-accumulate operations, respectively. This creates an "unbalanced" machine with respect to the vector memory bandwidth that increases the maximum performance of the machine for kernels with sufficient reuse.
```

Comment:
The comment on these configs being more suitable for higher-reuse kernels is good, but I think the term "unbalanced" has some negative connotations. Is there a better way to phrase this?